frontend/src/components/website/WebsiteForm.vue

```diff
@@ -0,0 +1,150 @@
+                                name="website"
+                                @change="changeName"
+                                v-model="name"
+                                :error-messages="nameErrors"
+                                single-line
+                                solo
+                                placeholder="website name"
+                                required
+                            />
+                        </div>
+                        <div class="inline-element">
+                            <label>Website Adress</label>
+                            <VTextField
+                                name="adress"
+                                placeholder="website adress"
+                                single-line
+                                solo
+                                disabled
+                                v-model="address"
+                            />
+                        </div>
+                        <div class="inline-element">
+                            <label>SPA</label>
+                            <VSwitch v-model="chips" />
+                        </div>
+                    </form>
+                    <div>
+                        <p class="inline-element">
+                            <span>Tracking ID: </span>
+                            <span>{{ trackingId }}</span>
+                        </p>
+                        <div>
+                            <h2>WebSite Tracking</h2>
+                            <p>
+                                This is Global Site tag tracking code for you website.
+                                Copy and Paste this code as the
+                                first item into the &lt;HEAD> of every Webpage you want to track
+                            </p>
+                            <VTextarea
+                                name="code"
+                                filled
+                                auto-grow
+                                readonly
+                                v-model="code"
+                                @focus="$event.target.select()"
+                                :value="code"
+                            />
+                        </div>
+                    </div>
+                </VContainer>
+            </VFlex>
+        </VContent>
+    </VApp>
+</template>
+
+<script>
+    import { mapActions, mapGetters } from 'vuex';
+    import { GET_WEBSITE, SET_NAME_WEBSITE } from "../../store/modules/website/types/actions";
+    import { GET_AUTHENTICATED_USER } from "../../store/modules/auth/types/getters";
+
+    export default {
+        data: () => ({
+            nameErrors: '',
+            chips: '',
+            trackingId: '',
+            name:'',
+            address:'',
+            code: '',
+        }),
+        created() {
+            this.website({
+                userId: this.user.id
+            }).then((data) => {
+                this.name = data.name;
```

Comment:
Better in your action save information about website in vuex, and then make getter and take info frim getter